packages/trpc/server/routers/loggedInViewer/outOfOffice.handler.ts

```diff
@@ -290,11 +357,32 @@ export const outOfOfficeEntryDelete = async ({ ctx, input }: TBookingRedirectDel
     throw new TRPCError({ code: "BAD_REQUEST", message: "out_of_office_id_required" });
   }
 
+  let deletingOooOfUserId = ctx.user.id;
+  let deletingOooOfEmail = ctx.user.email;
+  let deletingOooOfUserName = ctx.user.username;
```

Comment:
I don't like the variable names in the whole file; they need to be improved

Reply:
i was trying to be more descriptive for purpose of new variables, updated, hope it is better now.